pyrasterframes/src/main/python/docs/getting-started.pymd

```diff
@@ -43,15 +46,27 @@ You can also use RasterFrames in the following environments:
 
 ### Jupyter Notebook
 
-**TODO** User facing quick instructions e.g. how to pull and run docker hub hosted container
+ RasterFrames provides a Docker image for a Jupyter notebook server whose default kernel is already set up for running RasterFrames. To use it:
+
+ 1. Install [docker](https://docs.docker.com/install/)
+ 1. Pull the image: `docker pull s22s/rasterframes-notebook`
+ 1. Run a container with the image, for example: 
+
+      docker run -p 8808:8888 -p 44040:4040 -v /path/to/notebooks:/home/notebooks  rasterframes-notebook:latest
 
-See [RasterFrames Notebook README](https://github.com/locationtech/rasterframes/blob/develop/rf-notebook/README.md) for instructions on running a Jupyter notebook server within a Docker container that has a fully set up environment.
+ 1. In a browser, open `localhost:8808` in the example above. 
+
+See [RasterFrames Notebook README](https://github.com/locationtech/rasterframes/blob/develop/rf-notebook/README.md) for instructions on building the Docker image for this Jupyter notebook server.
 
 ### `pyspark` shell or app
 
-To initialize RasterFrames in a `pyspark` shell, prepare to call pyspark with the appropriate `--master` and other `--conf` arguments for your cluster manager and environment. To these you will add the PyRasterFrames assembly JAR and the python source zip.
+To initialize RasterFrames in a `pyspark` shell, prepare to call pyspark with the appropriate `--master` and other `--conf` arguments for your cluster manager and environment. To these you will add the PyRasterFrames assembly JAR and the python source zip. 
+
+You can either [build](https://github.com/locationtech/rasterframes/blob/develop/README.md) the artifacts or download them:
+
+ * Assembly JAR: https://repo1.maven.org/maven2/org/locationtech/rasterframes/pyrasterframes_2.11/${VERSION}/pyrasterframes-assembly-${VERSION}.jar
```

Comment:
This is a bit of a guess. I actually don't see the `assembly` jar published on maven central for the RC's

Reply:
also note that the artifact name is missing the scala version.

Reply:
Assemblies are *not* published on Maven Central... frowned upon.

We'll have to address the scala version tag later. Right now it's fine because it's not a published artifact, and gets added to the classpath outside the normal sbt chain.

Reply:
ah. so the instructions for this should read.. build with `sbt pyrasterframes/package` and then run this command?  

Or do we intend to publish the assembly jar in some other way?

Reply:
Hmm yes i guess that's another option. User could unzip the whl file. Should we have them do that?

Reply:
We will never publish the assembly outside of PyPi. Explain why we would need it?

Reply:
BTW, once you publish to maven central, you can use RasterFrames like this:

```
spark-shell --packages org.locationtech.rasterframes:rasterframes_2.11:0.8.0-RC4
```

Spark launcher handles getting all the transitive dependencies.

Reply:
If you want datasources:

```
org.locationtech.rasterframes:rasterframes-datasource_2.11:0.8.0-RC4
```

Reply:
I think we are talking past each other a little. 

I *think* the zip sources artifact plus the `--packages` may be the way to go.  

But we need to fix #233 

```suggestion

pyspark \
    --conf spark.serializer=org.apache.spark.serializer.KryoSerializer \
    --conf spark.kryo.registrator=org.locationtech.rasterframes.util.RFKryoRegistrator \
    --conf spark.kryoserializer.buffer.max=500m \
    --py-files pyrasterframes_2.11-${VERSION}-python.zip \
    --packages org.locationtech.rasterframes:rasterframes_2.11:${VERSION},org.locationtech.rasterframes:pyrasterframes_2.11:${VERSION},org.locationtech.rasterframes:rasterframes-datasource_2.11:${VERSION} 

```